test/sagemaker_tests/pytorch/training/integration/sagemaker/test_distributed_operations.py

```diff
@@ -30,6 +30,16 @@
 RESOURCE_PATH = os.path.join(os.path.dirname(__file__), '..', '..', 'resources')
 
 
+def validate_or_skip_fastai(ecr_image):
+    if not can_run_fastai(ecr_image):
+        pytest.skip("Fast ai is not supported on PyTorch v1.9 ")
```

Comment:
nit: We can just have this `if` condition in the `test_dist_operations_fastai_gpu` test function, right? There is no need to write it this way, because this does not need a complex set of checks like the SMDDP/SMDMP tests do.